lib/ruby-stackoverflow/client/resources.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'resources/resource'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.